### ISSUE TITLE
The information in the results is a bit cryptick

### ISSUE THREAD

**Issue by jjuarez84** [ggutierrez-bio/ligbinder#5]
It is not super clear why runs terminate. There is some info in the log file, but I think a more detailed output in the results will help gathering statistics to compare binders from non-binders. I don't have a super clear idea how that could work, happy to discuss at some point.

**Comment by ggutierrez-bio** [comment]
I agree, adding the stopping reason to the report could be interesting.
For comparison between binders and non-binders, i propose using:
* biasing power
* total number of nodes
* any other property related to how big the sampled space is
We should think of other relevant metrics
